Formula/mint.rb

```diff
@@ -1,8 +1,9 @@
 class Mint < Formula
   desc "Dependency manager that installs and runs Swift command-line tool packages"
   homepage "https://github.com/yonaskolb/Mint"
-  url "https://github.com/yonaskolb/Mint/archive/0.7.1.tar.gz"
-  sha256 "986dd967b818ee044ed0568135db8be2083ad6c3995fe6281ada9b42b224a2ec"
+  url "https://github.com/yonaskolb/mint/archive/0.8.0.tar.gz"
+  version "0.8.0"
```

Comment:
@yonaskolb This line is redundant since it comes from the `url`, did the tool add this automatically?